platform/autoupdate.services/src/org/netbeans/modules/autoupdate/services/Utilities.java

```diff
@@ -142,6 +142,9 @@ public static Collection<KeyStore> getKeyStore (TrustLevel trustLevel) {
      *
      * @param archiveCertPath
      * @param trustedCertificates
+     * @param trustedCACertificates
+     * @param validateCertificates
+     * @param validationCACertificates
      * @return
      */
```

Comment:
same here. The whole comment serves no purpose.

Reply:
fixed below